api/src/main/java/au/org/aodn/nrmn/restapi/service/GlobalLockService.java

```diff
@@ -0,0 +1,30 @@
+package au.org.aodn.nrmn.restapi.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import au.org.aodn.nrmn.restapi.data.model.GlobalLock;
+import au.org.aodn.nrmn.restapi.data.repository.GlobalLockRepository;
+
+@Service
+public class GlobalLockService {
+
+    @Autowired 
+    GlobalLockRepository globalLockRepository;
+
```

Comment:
Need to use transaction around this section to make read consistency?

Reply:
The save in the GlobalLockService is where the lock is acquired, and this method is atomic.
I'll add a `@Transactional` to the setLock method, but in practice it shouldn't matter.